Editor/EditorCore/EditorUtility.cs

```diff
@@ -292,16 +292,18 @@ internal static void MoveToActiveScene(GameObject gameObject)
         /// Initialize this object with the various editor-only parameters, and invoke the object creation callback.
         /// </summary>
         /// <param name="pb"></param>
-        internal static void InitObject(ProBuilderMesh pb)
+        internal static void InitObject(ProBuilderMesh pb, bool snapToGrid = true)
```

Comment:
`InitObject` is intended to apply the various user preferences, including snapping. Why are we passing `snapToGrid` instead of just checking if snapping is enabled?